### PR TITLE
ElastoplasticTorsion: Update to 9.1

### PR DESCRIPTION
**Pull request by tamiko**

### CODE REVIEW

**Comment by tamiko** [comment]
In reference to dealii/dealii#8080

File: ElastoplasticTorsion/ElastoplasticTorsion.cc [hunk 1]
```diff
@@ -763,10 +763,12 @@ namespace nsp
   template <int dim>
   void ElastoplasticTorsion<dim>::refine_mesh ()
   {
+    using FunctionMap = std::map<types::boundary_id, const Function<dim> *>;
+
```
Comment (bangerth) [inline ElastoplasticTorsion/ElastoplasticTorsion.cc]: Why do you need this? Don't we define this `typedef` in deal.II already?

Comment (tamiko) [reply ElastoplasticTorsion/ElastoplasticTorsion.cc]: I don't think so. The `FunctionMap` class template that had a `FunctionMap<dim>::type` declaration is deprecated. The corresponding Kelly error estimator functions use above `std::map` directly. Personally, I think that the `using FunctionMap = ...` is slightly more readible.

Comment (bangerth) [reply ElastoplasticTorsion/ElastoplasticTorsion.cc]: Did I see this wrong in the other PRs I just merged that you just used a bar `FunctionMap()` argument? Or was there a `using` declaration as well that I missed?

Comment (tamiko) [reply ElastoplasticTorsion/ElastoplasticTorsion.cc]: There was a `using` declaration as well :-)

**Pull request merged**